classes/class-p4-custom-taxonomy.php

```diff
@@ -164,42 +169,79 @@ public function register_taxonomy() {
 				'labels'       => $p4_page_type,
 				'show_ui'      => true,
 				'query_var'    => true,
-				'rewrite'      => [ 'slug' => 'page_type' ],
```

Comment:
I don't know if we want to keep this for any existing links maybe

This will replace the current rules from

```
| page_type/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$                          | index.php?p4-page-type=$matches[1]&feed=$matches[2]                          | p4-page-type |
| page_type/([^/]+)/(feed|rdf|rss|rss2|atom)/?$                               | index.php?p4-page-type=$matches[1]&feed=$matches[2]                          | p4-page-type |
| page_type/([^/]+)/embed/?$                                                  | index.php?p4-page-type=$matches[1]&embed=true                                | p4-page-type |
| page_type/([^/]+)/page/?([0-9]{1,})/?$                                      | index.php?p4-page-type=$matches[1]&paged=$matches[2]                         | p4-page-type |
| page_type/([^/]+)/?$                                                        | index.php?p4-page-type=$matches[1]                                           | p4-page-type |

```

to these

```
| p4-page-type/([^/]+)/feed/(feed|rdf|rss|rss2|atom)/?$                       | index.php?p4-page-type=$matches[1]&feed=$matches[2]                          | p4-page-type |
| p4-page-type/([^/]+)/(feed|rdf|rss|rss2|atom)/?$                            | index.php?p4-page-type=$matches[1]&feed=$matches[2]                          | p4-page-type |
| p4-page-type/([^/]+)/embed/?$                                               | index.php?p4-page-type=$matches[1]&embed=true                                | p4-page-type |
| p4-page-type/([^/]+)/page/?([0-9]{1,})/?$                                   | index.php?p4-page-type=$matches[1]&paged=$matches[2]                         | p4-page-type |
| p4-page-type/([^/]+)/?$                                                     | index.php?p4-page-type=$matches[1]                                           | 

```

Reply:
Hmmm, I'll check the routes more thoroughly, didn't catch these. Thanks!